docs/design/autoware-interfaces/ad-api/list/api/fail_safe/index.md

```diff
@@ -1,20 +1,41 @@
 # Fail-safe API
 
-- {{ link_ad_api('/api/fail_safe/state') }}
+- {{ link_ad_api('/api/fail_safe/mrm/state') }}
 
 ## Description
 
 This API manages the behavior related to the abnormality of the vehicle.
-Provides the state of Request to Intervene (RTI) and Minimum Risk Maneuver (MRM).
+It provides the state of Request to Intervene (RTI), Minimal Risk Maneuver (MRM) and Minimal Risk Condition (MRC).
+As shown below, Autoware has the gate to switch between the command during normal operation and the command during abnormal operation.
+For safety, Autoware switches the operation to MRM when an abnormality is detected.
+Since the required behavior differs depending on the situation, MRM is implemented in various places as a specific mode in a normal module or as an independent module.
+The fail-safe module selects the behavior of MRM according to the abnormality and switches the gate output to that command.
+
+![fail-safe-architecture](./architecture.drawio.svg)
 
 ## States
 
-![fail-safe-state](./state.drawio.svg)
+The MRM state indicates whether MRM is operating. This state also provides success or failure.
+Generally, MRM will switch to another behavior if it fails.
+
+![mrm-state](./mrm-state.drawio.svg)
+
+| State         | Description                                                |
+| ------------- | ---------------------------------------------------------- |
+| NORMAL        | The vehicle is normal. MRM is not operating.               |
```

Comment:
I personally feel `NONE` is better.

```suggestion
| NONE        | MRM is not operating.               |
```

Reply:
Then it seems better not to have the prefix `MRM_`. How about this? cc: @mkuri 
```
NONE
OPERATING
SUCCEEDED
FAILED
```

Reply:
I think it's okay.

Reply:
I think it's good.

Reply:
Done https://github.com/autowarefoundation/autoware-documentation/pull/165/commits/09be6c3f3ed80427bafe5529b33f3a0f1abf2325.